packages/curve-ui-kit/src/shared/ui/LargeTokenInput.tsx

```diff
@@ -281,23 +290,58 @@ export const LargeTokenInput = <T extends Amount>({
     maxBalance?.onMax?.call(null)
   }, [handlePercentageChange, maxBalance?.onMax])
 
+  const componentId = useId()
```

Comment:
wow I've never needed this before :grin: 
I think usually I'd add a class to the child and use child selectors, but this works too

Reply:
I didn't want to risk some parent accidentally have the same `className` as I really only want to target the specific parent. I guess either way works.